include/nuttx/fs/ioctl.h

```diff
@@ -177,6 +177,12 @@
                                            * OUT: None
                                            */
 
+#define FIOC_NOTIFY     _FIOC(0x000e)     /* IN:  Pointer to struct automount_notify_s
```

Comment:
FIOC_AUTOMOUNT_NOTIFY?

Reply:
Can we keep a short name?
I was just looking into
```
#define FIONREAD        _FIOC(0x0007)     /* IN:  Location to return value (int *)
                                           * OUT: Bytes readable from this fd
                                           */
#define FIONWRITE       _FIOC(0x0008)     /* IN:  Location to return value (int *)
                                           * OUT: Number bytes in send queue
                                           */
#define FIONSPACE       _FIOC(0x0009)     /* IN:  Location to return value (int *)
                                           * OUT: Free space in send queue.
                                           */
```
that are `FIONREAD` and not `FIOC_QUEUE_NREAD`, etc.

Reply:
Can we keep a short name?
I was just looking into
```
#define FIONREAD        _FIOC(0x0007)     /* IN:  Location to return value (int *)
                                           * OUT: Bytes readable from this fd
                                           */
#define FIONWRITE       _FIOC(0x0008)     /* IN:  Location to return value (int *)
                                           * OUT: Number bytes in send queue
                                           */
#define FIONSPACE       _FIOC(0x0009)     /* IN:  Location to return value (int *)
                                           * OUT: Free space in send queue.
                                           */
```
that are `FIONREAD` and not `FIOC_QUEUE_NREAD`, etc.

Reply:
Or maybe `FIOC_AM_NOTIFY`?

Reply:
I am fine with either.